frontend/src/libs/components/button/button.tsx

```diff
@@ -1,25 +1,34 @@
+import { Icon } from '~/libs/components/components.js';
+import { type IconType } from '~/libs/components/icon/common.js';
 import { getValidClassNames } from '~/libs/helpers/helpers.js';
 
 import styles from './styles.module.scss';
 
 type Properties = {
-  label: string;
-  size?: 'big' | 'small';
+  label?: string;
+  icon?: IconType;
+  size?: 'big' | 'small' | 'fitContent';
```

Comment:
```suggestion
  size?: 'big' | 'small';
```
handle it by class name